users/templates/users/profile.html

```diff
@@ -20,18 +37,69 @@ <h2 class="account-heading">{{ userProfile.username }}</h2>
         <p class="text-secondary">{{ userProfile.email }}</p>
       </div>
 
-      <table class="table table-hover">
-        <th>Signed Up Events</th>
-        {% for event in user.signed_up.all %}
+      <h3>Signed Up Events</h3>
+
+      <!-- Button trigger modal -->
+      <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#exampleModalLong">
+        Events Attending
+      </button>
+
+      <!-- Modal -->
+      <div class="modal fade" id="exampleModalLong" tabindex="-1" role="dialog" aria-labelledby="exampleModalLongTitle" aria-hidden="true">
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+
+            <div class="modal-header">
+              <h5 class="modal-title" id="exampleModalLongTitle"> Events List </h5>
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+              </button>
+            </div>
+
+            <div class="modal-body">
+
+              {% for event in user.signed_up.all %}
+                <a href="{% url 'event detail' event.id %}">
+                      {{ event.name }}
+                </a>
+                <br>
+              {% endfor %}
+            <div class="modal-footer">
+              <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+            </div>
+
+          </div>
+
+        </div>
+      </div>
+
+
+          <button type="button" class="btn btn-info" data-toggle="collapse" data-target="#demo">List of All Events </button>
+
+          {% for event in user.signed_up.all %}
+          <div id="demo" class="collapse">
+            <div>
+              <ul class="list-group">
+                <a href="{% url 'event detail' event.id %}">
+                  {{ event.name }}
+                </a>
+              </ul>
+            </div>
+          </div>
+          {% endfor %}
+
+
+
+        <!-- {% for event in user.signed_up.all %}
           <tr>
             <td>
                 <a href="{% url 'event detail' event.id %}">
                   {{ event.name }}
                 </a>
             </td>
           </tr>
-        {% endfor %}
-      </table>
+        {% endfor %} -->
+
 
     </div>
   </div>
```

Comment:
need one more </div> tag. The footer is getting centered on the profile page right now.